.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Bug description**
+A clear and concise description of what the bug is.
+
+**How To Reproduce**
+
```

Comment:
I think both input files/data **and** steps are important to reproduce an issue, so I'd suggest to add another list before the one below titled `Steps to reproduce the behavior`, something like this:
```
Steps to reproduce the behavior
  * Step 1
  * Step 2
  * ...
```